src/components/offer-card/OfferCard.jsx

```diff
@@ -0,0 +1,97 @@
+import { t } from 'i18next'
+
+import Rating from '@mui/material/Rating'
+import Avatar from '@mui/material/Avatar'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+import IconButton from '@mui/material/IconButton'
+import TurnedIn from '@mui/icons-material/TurnedIn'
+import TurnedInNot from '@mui/icons-material/TurnedInNot'
+
+import AppButton from '~/components/app-button/AppButton'
+import AppChip from '~/components/app-chip/AppChip'
+
+import { styles } from '~/components/offer-card/OfferCard.styles'
+
+const OfferCard = ({
```

Comment:
The card component should be created. It should be totally decomposed and not be aware of name bio rating, it should just some fileds title description etc, maybe children if the content is too complicated. Then Wrapper from Offer Card should be created on it base.